mitiq/cdr/cdr_tests.py

```diff
@@ -0,0 +1,121 @@
+import cirq
```

Comment:
See https://github.com/unitaryfund/mitiq/blob/master/mitiq/zne/tests/test_inference.py for an example of unit testing, and https://github.com/unitaryfund/mitiq/blob/master/mitiq/zne/inference.py for the file being tested.

Here, the test file should be `tests/test_clifford_training_data.py`.